assets/js/components/ClusterDetails/ChecksResults.jsx

```diff
@@ -47,24 +47,8 @@ const sortHosts = (hosts = []) => {
   });
 };
 
-const getResultIcon = (checks_execution, result) => {
-  switch (checks_execution) {
-    case 'requested':
-      return <EOS_SCHEDULE className="fill-gray-500" />;
-    case 'running':
-      return <Spinner></Spinner>;
-  }
-
-  switch (result) {
-    case 'passing':
-      return <EOS_LENS_FILLED className="fill-jungle-green-500" />;
-    case 'warning':
-      return <EOS_LENS_FILLED className="fill-yellow-500" />;
-    case 'critical':
-      return <EOS_LENS_FILLED className="fill-red-500" />;
-    default:
-      return <EOS_LENS_FILLED className="fill-gray-500" />;
-  }
+const getResultIcon = (execution_state, health) => {
+  return <ExecutionIcon health={health} execution_state={execution_state} />;
```

Comment:
For what I know usually in the `js` ecosystem `camelCase` is preferred to `snake_case`, and I also prefer it :smile: 

I am aware we are actually using `snake_case` somewhere else in the fronted codebase, and that's mainly because the api responses are snake case (see redux states, sagas and basically whenever we use json data from apis).

If we can use camelCase here, it'd be preferable.